realm/realm-library/src/syncIntegrationTest/java/io/realm/PermissionManagerTests.java

```diff
@@ -54,6 +55,7 @@
 import static org.junit.Assert.fail;
 
 @RunWith(AndroidJUnit4.class)
+@Ignore // FIXME: Temporary disable unit tests due to lates (3.0.0-alpha.2) ROS having issues
```

Comment:
Probably shouldn't be committed, I'm guessing?

Reply:
Actually, this is intentional to unblock CI, but we probably shouldn't release anything until we determined what is causing this.